Lib/test/test_subprocess.py

```diff
@@ -3332,12 +3345,23 @@ def test_startupinfo_copy(self):
             self.assertEqual(startupinfo.lpAttributeList, {"handle_list": []})
 
     def test_creationflags(self):
-        # creationflags argument
-        CREATE_NEW_CONSOLE = 16
         sys.stderr.write("    a DOS box should flash briefly ...\n")
         subprocess.call(sys.executable +
                         ' -c "import time; time.sleep(0.25)"',
-                        creationflags=CREATE_NEW_CONSOLE)
+                        creationflags=_winapi.CREATE_NEW_CONSOLE)
```

Comment:
I don't know why subprocess references `_winapi` for constants such as `STARTF_USESHOWWINDOW` and `SW_HIDE`, since they're already imported as globals. Anyway, surely at least here you can use `subprocess.CREATE_NEW_CONSOLE` and not worry about importing `_winapi`.

Reply:
Done and tests still pass, thanks!